src/lib/core/components/filter/TableFilter.tsx

```diff
@@ -318,7 +322,7 @@ export function TableFilter({
         tableSummary.value.variableId === variable.id && (
           <MembershipField
             displayName={entity.displayName}
-            dataCount={tableSummary.value.entitiesCount}
+            dataCount={totalEntityCount}
```

Comment:
I'm actually wondering if the following line should be. It would seem to make sense. Hard to know for sure, because MembershipField doesn't seem to use this prop.

```
filteredDataCount={filteredEntityCount}
```

Reply:
It should be the background count. It is used here: https://github.com/VEuPathDB/WDKClient/blob/028c0e3472b5a002ebc5a90a9bd70050011aa37d/Client/src/Components/AttributeFilter/MembershipField.jsx#L492

Reply:
Sorry I was talking about line 326.  I wasn't clear.

`MembershipField.filteredDataCount` doesn't seem to be used except in [FilterList](https://github.com/VEuPathDB/WDKClient/blob/1d179fe1825dac38134273fa0e1d8de670e96f22/Client/src/Components/AttributeFilter/FilterList.jsx#L92) which is not used in EDA.

But for symmetry I think it should be `filteredDataCount={filteredEntityCount}`

Reply:
Oh, I see. Sorry, that line didn't show up in the review section. Can you remind me why `tableSummary.value.entitiesCount` isn't good here?